README.md

```diff
@@ -185,6 +185,23 @@ describe 'myclass' do
 end
 ```
 
+```ruby
```

Comment:
I added the example here just so you could easily compare it to the block above it.